src/Sulu/Component/Webspace/Loader/XmlFileLoader10.php

```diff
@@ -492,8 +492,7 @@ protected function generateEnvironments(\DOMNode $portalNode, Portal $portal)
     {
         foreach ($this->xpath->query('x:environments/x:environment', $portalNode) as $environmentNode) {
             /** @var \DOMNode $environmentNode */
-            $environment = new Environment();
-            $environment->setType($environmentNode->attributes->getNamedItem('type')->nodeValue);
+            $environment = new Environment($environmentNode->attributes->getNamedItem('type')->nodeValue);
```

Comment:
The ->nodeValue of the XML Node actually has type `string|null` So there is no BC break?